### PR TITLE
5 1 source and election

### PR DESCRIPTION
**Pull request by sduckett**
This adds a page that displays the interesting parts of source and election elements from a feed. Most of the interesting work is in this *blocking* [data-processor PR](https://github.com/votinginfoproject/data-processor/pull/241)

[Pivotal card](https://www.pivotaltracker.com/story/show/128176637)

screen shot 2016-12-02 at 3 13 08 pm

### CODE REVIEW

File: public/app/partials/5.1/aside.html [hunk 1]
```diff
@@ -20,7 +20,8 @@
     <nav class="feed-nav">
       <ul>
         <li><a id="sidebar-overview" class="is-finished" href="{{feedURL('')}}">Overview</a></li>
-        <li><a id="sidebar-source" class="is-finished" href="{{feedURL('/errors')}}">Errors</a></li>
+        <li><a id="sidebar-source" class="is-finished" href="{{feedURL('/source')}}">Source & Election</a></li>
```
Comment (tank157) [inline public/app/partials/5.1/aside.html]: Could consider html escaping the `&` here. Probably not a huge deal.

File: public/app/partials/5.1/source.html [hunk 2]
```diff
@@ -0,0 +1,98 @@
+<p ng-if="source.error_count !== undefined">
```
Comment (tank157) [inline public/app/partials/5.1/source.html]: Does this work if `source` is not yet defined? It seems like the code below in the loading and data module sections is checking at that level.

Comment (tank157) [inline public/app/partials/5.1/source.html]: Similar question with election error count further down.

File: public/app/partials/5.1/aside.html (hunk 1 above)
Comment (sduckett) [inline public/app/partials/5.1/aside.html]: Good idea!

File: public/app/partials/5.1/source.html (hunk 2 above)
Comment (sduckett) [inline public/app/partials/5.1/source.html]: [In the controller](https://github.com/votinginfoproject/Metis/pull/345/files#diff-db1a977b0c3b8acc636d22275df68040R11) we add these entities to our `$scope`.

**Pull request merged**